addons/player-thumb/addon.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Show project thumbnail under green flag overlay",
+  "description": "Shows the project's actual thumbnail instead of its initial state before starting the project.",
+  "credits": [
+    {
+      "name": "Samq64",
+      "link": "https://scratch.mit.edu/users/samq64/"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": ["projects"]
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"],
+      "runAtComplete": false
+    }
+  ],
+  "dynamicEnable": true,
+  "dynamicDisable": true,
+  "versionAdded": "1.28.0",
+  "tags": ["community", "theme"],
```

```suggestion
  "tags": ["community", "projectPage"],
```
In my opinion, this is more of a behavior change than a theme. There's only one thing that's having its appearance changed, and the content is dynamic.